docs/governance/tokenomics.md

```diff
@@ -2,200 +2,178 @@
-
-This could be the case if someone duplicates their validator and try to double their voting power.
+:::caution 
+Slashing can still occur during the unbonding period if there is evidence of bad behaviour during the period when a Validator was bonded.
+:::
 
-Slashed tokens are sent to the community pool.
+### Token holder delegating
 
+Anyone holding CUDOS tokens can delegate tokens to one or more **validator nodes** of their choice. Validators and Delegators then share rewards **based on the amounts they have contributed to the pool**.
 
-## Transactions 
+Validator Operators are free to set their own **commission rate** to charge for delegating. The **commission rate** can change at most once daily. However, a validator's **commission rate** CANNOT exceed the **maximum commission rate** set when the validator node was first initiated. Commission fees can be viewed under [Validator details in the Cudos Explorer](https://explorer.cudos.org/validators). 
 
-How to view the mempool? Or pending transactions 
-If you're a validator you can keep your mempool to yourself when it is your turn to propose a block then you can submit your own transactions. 
+![validator-details](@site/static/img/validator-details.png)
 
-If you sync your own node with other nodes then you can view pending transactions.
+### Redelegation
 
-* Tx fees are calculated based on the **gas_wanted value rather than gas_used**. Therefore users need to be aware of this and should not provide extra gas_wanted values “just in case”.
+1. A **Delegator** can **redelegate** tokens from **Validator A** to **Validator B** without incurring any penalties or extra waiting times. A redelegation maturation period now follows for 21 days. 
 
-* We have a min-gas-price parameter set at 0.000005 CUDOS below which txs won’t be accepted into mempools
+2. Following on from the above example, if a **Delegator** wishes to **redelegate** their tokens from **Validator B** to a hypothetical **Validator C** they must wait *21 days*.
 
-    This parameter is local to each node on the network, can be changed off-chain, and cannot be queried on-chain.
+3. A **Validator Operator** can redelegate funds to an *existing* node in two ways:
 
-    Hence we must rely on the community for this implementation to work properly.
+- Connecting a wallet to the [**Cudos Dashboard**](https://dashboard.cudos.org/staking) and redelegating funds to a Validator node of their choice. 
 
-* Full-nodes that keep these mempools must have a mechanism to protect themselves from over-usage.
+- Using `cudos-noded` CLI to redelegate funds. 
 
-:::warning 
-Default `gas-wanted on the CLI is 200,000`. Hence the fee is calculated based on this. Users must be aware of this. Be careful to specify this as what you are prepared to pay. 
+:::caution
 
-Hence most transactions cost 1 CUDOS
+The delegated stake on a Validator node must be at least 2M CUDOS to continue running
```

Comment:
I think this is connected to item #3, which I've just suggested to remove :) So I'd suggest to remove this tip as well. However, if you want to keep it, please note that it is the self-delegation amount that needs to be more than 2M. Besides, this is a bit tricky because the 2M is only the default value. A validator can choose to increase it, but not decrease it. It is seen as a level commitment. Hence if a validator has increased it to, for example, 3M, then their self-delegation needs to be above 3M in order to continue running.

Reply:
done